gpAux/gpperfmon/src/gpmon/gpperfmon_install

```diff
@@ -164,35 +165,63 @@ if __name__ == '__main__':
         cmd = Command("""PGPORT=%d psql template1 -c "DROP ROLE IF EXISTS gpmon" """ % options.port)
         commands.append(cmd)
 
-        cmd = Command("""PGPORT=%d psql template1 -c "CREATE ROLE gpmon WITH SUPERUSER CREATEDB LOGIN ENCRYPTED PASSWORD '%s'" """ % (options.port, options.password))
+        if options.password:
+            cmd = Command("""PGPORT=%d psql template1 -c "CREATE ROLE gpmon WITH SUPERUSER CREATEDB LOGIN ENCRYPTED PASSWORD '%s'" """ % (options.port, options.password))
+        else:
+            cmd = Command("""PGPORT=%d psql template1 -c "CREATE ROLE gpmon WITH SUPERUSER CREATEDB LOGIN" """ % options.port)
         commands.append(cmd)
 
-        cmd = Command("""echo "local    gpperfmon         gpmon         md5" >> %s""" % pg_hba, showOutput=True)
-        commands.append(cmd)
+        if options.password:
+            cmd = Command("""echo "local    gpperfmon         gpmon         md5" >> %s""" % pg_hba, showOutput=True)
+            commands.append(cmd)
 
-        cmd = Command("""echo "host     all         gpmon         127.0.0.1/28    md5" >> %s""" % pg_hba, showOutput=True)
-        commands.append(cmd)
+            cmd = Command("""echo "host     all         gpmon         127.0.0.1/28    md5" >> %s""" % pg_hba, showOutput=True)
+            commands.append(cmd)
 
-        cmd = Command("""echo "host     all         gpmon         ::1/128    md5" >> %s""" % pg_hba, showOutput=True)
-        commands.append(cmd)
+            cmd = Command("""echo "host     all         gpmon         ::1/128    md5" >> %s""" % pg_hba, showOutput=True)
+            commands.append(cmd)
+        else:
+            cmd = Command("""echo "local    gpperfmon         gpmon                         ident        map=perfmon" >> %s""" % pg_hba, showOutput=True)
```

Comment:
ident works only over TCP, not UNIX socket
```suggestion
            cmd = Command("""echo "local    gpperfmon         gpmon                         peer        map=perfmon" >> %s""" % pg_hba, showOutput=True)
```

Reply:
update PR description too

Reply:
right, done